app/src/main/java/com/osfans/trime/data/theme/mapper/ToolBarMapper.kt

```diff
@@ -0,0 +1,59 @@
+// SPDX-FileCopyrightText: 2015 - 2025 Rime community
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+package com.osfans.trime.data.theme.mapper
+
+import com.charleskorn.kaml.YamlMap
+import com.charleskorn.kaml.YamlScalar
+import com.osfans.trime.data.theme.model.ToolBar
+
+class ToolBarMapper(node: YamlMap) : Mapper<ToolBar>(node) {
+
+    override fun map(): ToolBar = ToolBar(
```

Comment:
If we use the format as I suggest above, we can save the trouble of this kind of custom mapping.